src/Radicle/Ipfs.hs

```diff
@@ -42,11 +42,13 @@ module Radicle.Ipfs
 import           Protolude hiding (TypeError, catch, catches, try)
 
 import           Control.Exception.Safe
```

Comment:
Maybe make the unqualified exports explicit then?

Reply:
Like this ea987dc0bc21a296369e86fdba351501ca6ca90f?

I used the qualified `Safe.onException` to make the semantics wrt `skipAsyncExceptions` more explicit. If you're using `safe-exceptions` everywhere anyways, that's maybe not necessary.